.github/workflows/msrv.yml

```diff
@@ -0,0 +1,33 @@
+name: Check MSRV
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "**.rs"
+      - "**/Cargo.toml"
+  pull_request:
```

Comment:
Could this be simplified using the following syntax? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-multiple-events

Reply:
Given the yaml syntax, that does not seem possible.

Reply:
Yeah it seems that you're right and there's no elegant way to avoid duplication here. https://stackoverflow.com/a/63581747/2148718